Build Failure: WrongConstant Error in ReactContextExtensions.kt on Lint Check

### Description

After installing react-native-gesture-handler and running ./gradlew clean build, the build fails due to a lint error in ReactContextExtensions.kt. The error message indicates that UIManagerType.FABRIC is not recognized as a valid constant, causing the lint check to fail.

**Error Log:**

`Task :react-native-gesture-handler:lintDebug FAILED

MobileApp/node_modules/react-native-gesture-handler/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt:10: Error: Must be one of: UIManagerType.DEFAULT, UIManagerType.FABRIC [WrongConstant]
  val fabricUIManager = UIManagerHelper.getUIManager(this, UIManagerType.FABRIC) as FabricUIManager
                                                           ~~~~~~~~~~~~~~~~~~~~

   Explanation for issues of type "WrongConstant":
   Ensures that when parameter in a method only allows a specific set of
   constants, calls obey those rules.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-gesture-handler:lintDebug'.
> Lint found errors in the project; aborting build.
Expected Behavior:
The build should complete successfully without lint errors.`

**Additional Context:**
This issue seems to be related to UIManagerHelper.getUIManager(), potentially due to API changes in React Native 0.78.1.

### Steps to reproduce

1. Install react-native-gesture-handler in a React Native project (0.78.1).

2. Run ./gradlew clean build.

3. Observe the lint failure.

### Snack or a link to a repository

none

### Gesture Handler version

2.24.0

### React Native version

0.78.1

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

Fabric (New Architecture)

### Build type

None

### Device

None

### Device model

_No response_

### Acknowledgements

Yes

--- a/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
+++ b/android/fabric/src/main/java/com/swmansion/gesturehandler/ReactContextExtensions.kt
@@ -5,8 +5,10 @@ import com.facebook.react.fabric.FabricUIManager
 import com.facebook.react.uimanager.UIManagerHelper
 import com.facebook.react.uimanager.common.UIManagerType
 import com.facebook.react.uimanager.events.Event
+import android.annotation.SuppressLint
 
 fun ReactContext.dispatchEvent(event: Event<*>) {
+  @SuppressLint("WrongConstant")
   val fabricUIManager = UIManagerHelper.getUIManager(this, UIManagerType.FABRIC) as FabricUIManager
   fabricUIManager.eventDispatcher.dispatchEvent(event)
 }

